test/core/xds/xds_client_fuzzer.cc

```diff
@@ -225,6 +225,14 @@ class Fuzzer {
     return &bootstrap.server();
   }
 
+  void DumpClientConfigBinary() {
+    upb::Arena arena;
+    auto client_config = envoy_service_status_v3_ClientConfig_new(arena.ptr());
+    std::vector<std::unique_ptr<std::string>> string_pool;
+    MutexLock lock(xds_client_->mutex());
```

Comment:
Suggest using a test spouse pattern to get access to the protected methods here.

Reply:
My first attempy at using this pattern, please review.